config/config.go

```diff
@@ -357,6 +357,7 @@ type Validations struct {
 type KialiFeatureFlags struct {
 	IstioInjectionAction bool        `yaml:"istio_injection_action,omitempty" json:"istioInjectionAction"`
 	IstioUpgradeAction   bool        `yaml:"istio_upgrade_action,omitempty" json:"istioUpgradeAction"`
+	SkipSubsetValidation bool        `yaml:"skip_subset_validation,omitempty" json:"skipSubsetValidation"`
```

Comment:
this should be moved under the "Validations" struct since it is a feature flag specifically involving validations.